### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by xiaoshier-good** [ph-w2000/DuoCast#5]
Hello, this paper has a huge workload and is a very good one. However, when conducting experiments on the Shanghai dataset, I encountered the following issues:
1. What is the appropriate training epoch?
2. Is CSI-M the evaluation criterion for selecting the best weight?

**Comment by ph-w2000** [comment]
Thank you for your interest in our work.

The appropriate training epoch depends on the batch size and learning rate. In practice, the appropriate training epoch is selected based on validation performance. We usually train the model until the validation metrics become stable, and then select the checkpoint with the best validation performance.
Yes, CSI-M is the main evaluation criteria for selecting the best weight. In some cases, we also consider SSIM together with CSI-M, especially when two checkpoints have very close CSI-M values, to ensure the selected model also preserves reasonable spatial structure.

**Comment by xiaoshier-good** [comment]
This paper has a high workload and is a very good one. I encountered the following problems while reproducing this paper, and I hope you can provide me with help&nbsp;
Firstly：on the CIKM dataset, I chose the H5 file provided in the DiffCast paper. However, I noticed that you set the Pixel_Scale to 90 in the dataset_cikm.py code file. Although this setting is the same as in the DiffCast paper, it is different from the one set in the AlphaPre paper, which is 80. I am curious about how you compared the evaluation metrics in Table 2 at the end. What is the value you set on the CIKM dataset?
Secondly：can I directly run the CIKM dataset experiment in this paper using the processed H5 file provided by DiffCast?
Thirdly：
CUDA_VISIBLEDEVICES=0,1,2,3 accelerate launch\
--num_processes 4 \
run.py \
--dataset cikm \
--data_path data/CIKM2017/cikm.h5 \
--seq_len 15 \
--frames_in 5 \
--frames_out 10 \
--train_stage stage1 \
--batch_size 4 \
--num_workers 8 \
--exp_dir cikm_full_new \
--exp_note stage1_new \
--epochs 50 \
--lr 1e-4 \
--theta_int 35 \
Why does the data generated by Stage 1 in the first stage always not match your paper's metrics, and even have a significant gap. Although I know that the data from stage 1 cannot be compared with this paper in essence, the gap is significant. Can you provide the H5 file of CIKM and the terminal instructions/or weight configuration for the first stage of the CIKM dataset. Thank you!




2122397417xt
***@***.***
